packages/swapper/src/swappers/CowSwapper/utils/constants.ts

```diff
@@ -21,23 +21,7 @@ export const SUPPORTED_CHAIN_IDS: ChainId[] = [
   KnownChainIds.EthereumMainnet,
   KnownChainIds.GnosisMainnet,
   KnownChainIds.ArbitrumMainnet,
-].filter(chainId => {
-  if (
-    process.env['REACT_APP_FEATURE_COWSWAP_GNOSIS'] !== 'true' &&
```

Comment:
This was always `undefined` on boot when the constants were loaded because `process.env` was `{}`.